package.json

```diff
@@ -49,23 +50,29 @@
     "@evilmartians/lefthook": "^1.2.2",
     "@react-native-community/eslint-config": "^3.0.2",
     "@release-it/conventional-changelog": "^5.0.0",
-    "@types/jest": "^28.1.2",
-    "@types/react": "~17.0.21",
-    "@types/react-native": "0.70.0",
+    "@testing-library/react-native": "^11.5.2",
+    "@types/jest": "^29.2.5",
+    "@types/react": "^18.0.26",
+    "@types/react-native": "^0.70.0",
```

Comment:
Since our `peerDependencies` is
```
  "peerDependencies": {
    "react": "*",
    "react-native": "*"
  },
```
Please check if using this version of `@type` is not causing error

Reply:
Move to different ticket